opm/simulators/flow/FIBlackoilModel.hpp

```diff
@@ -74,7 +74,7 @@ class FIBlackOilModel : public BlackOilModel<TypeTag>
 public:
     explicit FIBlackOilModel(Simulator& simulator)
         : BlackOilModel<TypeTag>(simulator)
-        , element_chunks_(this->gridView_, ThreadManager::maxThreads())
+        , element_chunks_(this->gridView_, Dune::Partitions::all, ThreadManager::maxThreads())
```

Comment:
long line

Reply:
I have split it, but this one was 94 characters...